docs/crocks/Either.md

```diff
@@ -10,80 +10,84 @@
 --
 ```
 
-`Either` exposes the following constructor and instances:
+`Either` exposes the following functions on the constructor and instance:
 
 | Constructor | Instance |
 |:---|:---|
-| [`Left`](#left), [`Right`](#right), [`of`](#of) | [`inspect`](#inspect), [`value`](#value), [`type`](#type), [`equals`](#equals), [`map`](#map), [`ap`](#ap), [`of`](#of), [`chain`](#chain), [`sequence`](#sequence), [`traverse`](#traverse) |
+| [`Left`](#left), [`of`](#of), [`Right`](#right), [`type`](#type) | [`alt`](#alt), [`ap`](#ap), [`bimap`](#bimap), [`chain`](#chain), [`coalesce`](#coalesce), [`either`](#either), [`equals`](#equals), [`inspect`](#inspect), [`map`](#map),  [`of`](#of), [`sequence`](#sequence), [`swap`](#swap), [`traverse`](#traverse), [`type`](#type) |
```

Comment:
What do you think about maybe doing the tagged construction functions first, in a left to right order.
then `of` if it is there, and end with `type`.

So here it would be `Left`, `Right`, `of`, `type`.

And then keep the instance functions alphabetized?